docs/content/latest/explore/ysql-language-features/gin.md

```diff
@@ -0,0 +1,501 @@
+# Generalized inverted indexes
+
+In YugabyteDB, tables and secondary indexes are both key-value stores.
```

Comment:
nit: I think this can be misleading to regular users. Maybe you can say something like are "internally stored as" or something like that.